utils/build-presets.ini

```diff
@@ -708,7 +708,7 @@ install-llbuild
 install-swiftpm
 install-xctest
 install-prefix=/usr
-swift-install-components=autolink-driver;compiler;clang-builtin-headers;stdlib;swift-remote-mirror;sdk-overlay;license
+swift-install-components=autolink-driver;compiler;clang-builtin-headers;stdlib;swift-remote-mirror;sdk-overlay;license;sourcekit-inproc
```

Comment:
Oh. I didn't notice. It's a leftover of one of my cherrypicks from Nomura's PR (#8485). Helped me test things in linux.
It can, and probably should, be removed... even though that PR has already been aprooved. You can do it if you want, or just wait for me (atm I'm not at the computer) :)